packages/docs/src/components/global/DocRoadmapCollaborateBtn.vue

```diff
@@ -0,0 +1,20 @@
+<template>
+	<VBtn
+		outlined
+		color="primary"
+		class="text-none"
+		:href="href"
+	>
+		Soumettre une demande
+	</VBtn>
+</template>
+
+<script lang="ts">
+	import Vue from 'vue';
+	import Component from 'vue-class-component';
+
+	@Component
+	export default class DocRoadmapSubmitBtn extends Vue {
```

Comment:
```suggestion
	export default class DocRoadmapRequestBtn extends Vue {
```
Qu'est-ce que tu penses de renommer le composant en `DocRoadmapRequestBtn ` ? 🤔

Reply:
Ah oui parfait, je trouvais pas le bon nom